### PR TITLE
update base images

### PR DESCRIPTION
**Pull request by matancarmeli7**
Signed-off-by: matancarmeli7 <matan.carmeli7@gmail.com>

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -25,8 +25,7 @@ COPY pkg/ pkg/
 RUN CGO_ENABLED=0 GOOS=${TARGETOS:-linux} GOARCH=${TARGETARCH} go build -a -o manager main.go
 
 # Use distroless as minimal base image to package the manager binary
```
Comment (ArbelNathan) [inline Dockerfile:27]: remove too?

Comment (matancarmeli7) [reply Dockerfile]: done

**Review by ELENAGER — DISMISSED** [review]
LGTM

**Review by ArbelNathan — DISMISSED** [review]
LGTM

**Review by ELENAGER — APPROVED** [review]
LGTM

**Pull request merged**